frontend/src/modules/settings/containers/InfoTab.tsx

```diff
@@ -36,9 +41,17 @@ const useStyles = makeStyles((theme: Theme) => ({
 }));
 
 /**
- * Props
+ * @typedef Props
+ *
+ * some description.
```

Comment:
This placeholder needs to be filled in. For example, is this used for providing values to a component to render?

Reply:
Yes it provides system info value to render in UI. I am updating the docs with following comments
```
/**
 * @typedef Props
 *
 * System info data shown in UI
 * /
```